classes/ActionScheduler_Versions.php

```diff
@@ -115,6 +115,58 @@ public static function initialize_latest_version() {
 	 * @return string
 	 */
 	public function active_source() {
+		$file   = __FILE__;
+		$dir    = __DIR__;
+		$path   = $this->active_source_path();
```

Comment:
I may be mistaken, but I don't believe `$path` is used, so it can probably be removed.